model/strand.rb

```diff
@@ -79,7 +80,8 @@ def load(snap = nil)
   def unsynchronized_run
     start_time = Time.now
     prog_label = "#{prog}.#{label}"
-    Clog.emit("starting strand") { {strand: values, strand_started: {delay: start_time - schedule, prog_label: prog_label}} }
+    delay = @pre_lease_schedule ? start_time - @pre_lease_schedule : nil
+    Clog.emit("starting strand") { {strand: values, strand_started: {delay: delay, prog_label: prog_label}} }
```

Comment:
great find! I'm wondering if we should log delay here
`Clog.emit("obtained lease") { {lease_acquired: {time: lease_time}} }` 

and potentially even get rid of this log line. 

that way we also wouldn't need `@pre_lease_schedule`. but it's definitely an improvement to the current broken log output, so I'm also ok to merge as is.

Reply:
Actually it's a good suggestion. I moved the delay property from 'strand_started' to 'lease_acquired'. We run the strand for 30 seconds when we acquire the lease. Therefore, delays become meaningful when we first start running the strand. It's not meaningful for consecutive runs since we only take schedule into account when take the lease.